_data/2019/capetown/speakers.yml

```diff
@@ -2,14 +2,24 @@
+  bio: "Matthew Salganik is Professor of Sociology at Princeton University, and he is affiliated with several of Princeton's interdisciplinary research centers: the Office for Population Research, the Center for Information Technology Policy, the Center for Health and Wellbeing, and the Center for Statistics and Machine Learning. His research interests include social networks and computational social science. He is the author of the book *Bit by Bit: Social Research in the Digital Age.*"
   website: https://www.princeton.edu/~mjs3/
 
 - name: Chris Bail
   image: /assets/images/chrisnew.jpg
   bio: "Chris Bail is the Douglas and Ellen Lowey Associate Professor of Sociology and Public Policy at Duke University and a member of the Interdisciplinary Program on Data Science, the Duke Network Analysis Center, and the Duke Population Research Institute. His research examines how non-profit organiations and other political actors shape social media discourse using large text-based datasets and apps for social science research. He is the author of *Terrified: How Anti-Muslim Fringe Organizations Became Mainstream*." 
   website: http://www.chrisbail.net/
 
+- name: Marshini Chetty
+  image: /2018/capetown/images/Marshini.jpg
+  bio: "Marshini Chetty is a research scholar in the Department of Computer Science at Princeton University where she directs the Princeton Human Computer Interaction laboratory. She specializes in human computer interaction, usable security, and ubiquitous computing. She has a Ph.D. in Human-Centered Computing from Georgia Institute of Technology, USA and a Masters and Bachelors in Computer Science from University of Cape Town, South Africa. Marshini regularly publishes at top tier venues including CHI, CSCW, and SOUPS."
+  website: https://www.marshini.net/
+  
+ - name: Nick Feamster
+  image: /2018/capetown/images/Feamster.png
+  bio: "Nick Feamster is a professor in the Computer Science Department at Princeton University and the Deputy Director of the Princeton University Center for Information Technology Policy (CITP). Before joining the faculty at Princeton, he was a professor in the School of Computer Science at Georgia Tech. He received his Ph.D. in Computer science from MIT in 2005, and his S.B. and M.Eng. degrees in Electrical Engineering and Computer Science from MIT in 2000 and 2001, respectively. His research focuses on many aspects of computer networking and networked systems, with a focus on network operations, network security, and censorship-resistant communication systems."
+  website: https://www.cs.princeton.edu/~feamster/
```

Comment:
I think you might have to put this website in quotation marks -- might be the reason why the test is failing?